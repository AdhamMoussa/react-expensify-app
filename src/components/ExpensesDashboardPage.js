import React from 'react';
import ExpensesList from './ExpensesList';

const ExpensesDashboardPage = (props) => {
  return (
    <div>
      <h2>Your Expenses</h2>
      <ExpensesList />
    </div>
  )
};


export default ExpensesDashboardPage;
