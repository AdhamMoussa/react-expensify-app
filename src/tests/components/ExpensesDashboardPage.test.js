import React from 'react';
import { shallow } from 'enzyme';
import ExpensesDashboardPage from '../../components/ExpensesDashboardPage';

test('should render ExpensesDashboardPage correctly', () => {
  const wrapper = shallow(<ExpensesDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
