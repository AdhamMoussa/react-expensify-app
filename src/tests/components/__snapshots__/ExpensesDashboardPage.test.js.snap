// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesDashboardPage correctly 1`] = `
<div>
  <h2>
    Your Expenses
  </h2>
  <Connect(ExpensesList) />
</div>
`;
